zulipterminal/model.py

```diff
@@ -520,6 +542,9 @@ def append_message(self, event: Event) -> None:
                 if recipients == msg_recipients:
                     self.msg_list.log.append(msg_w)
 
+            if(not dummy_message_found):
```

Comment:
Other than the unnecessary parentheses, what is this for?

Reply:
So this is as per conversation https://chat.zulip.org/#narrow/stream/206-zulip-terminal/topic/PR.20.23T278/near/696566. We don't want new messages popping up in same narrow. ( This doesn't delete the message, only avoids displaying it)